core/sql/exp/ExpLOBaccess.h

```diff
@@ -515,7 +515,7 @@ class ExLob
 
   public:
 
-    char lobDataFile_[MAX_LOB_FILE_NAME_LEN];
+    string lobDataFile_; // TODO: change to NAString when ExLobCursor is allocated off of an NAHeap
```

Comment:
Can this not be allocated from NAString now  ? The ExLob class which contains this is allocated from the executor heap.